ckanext/saml2auth/plugin.py

```diff
@@ -112,7 +112,10 @@ def logout(self):
             log.info(u'User {0}<{1}> logged out successfully'.format(g.userobj.name, g.userobj.email))
         else:
             log.info(u'No user was logged in!')
-
+        
+        if toolkit.check_ckan_version("2.10"):
```

Comment:
This will check only for 2.10 but we want to work for 2.11 too

Reply:
@tino097 You got a point there. I change it to check if min CKAN version is 2.10. Can You please check if it's OK now